libs/community/langchain_community/chat_models/perplexity.py

```diff
@@ -228,10 +228,12 @@ def _stream(
                 chunk = chunk.dict()
             if len(chunk["choices"]) == 0:
                 continue
+            citations = chunk.get("citations", [])
```

Comment:
i believe citations come back with every chunk, so you'll have to only return them on the first chunk, or otherwise detect that the citations are identical to the ones already outputted.

Otherwise citations will be repeated when merging AIMessageChunk objects